src/components/Sidebar.js

```diff
@@ -1,25 +1,37 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
-import Drawer from 'material-ui/Drawer'
-import logo from '../images/logo.svg'
-import LogoutLink from './LogoutLink'
+import React from 'react';
+import { withStyles } from 'material-ui/styles';
 
-/*
-  Renders the IAR application side bar.
+import LogoutLink from './LogoutLink';
+import Divider from 'material-ui/Divider';
+import Toolbar from 'material-ui/Toolbar';
+import Typography from 'material-ui/Typography';
+import List from 'material-ui/List';
+import SidebarNavLink from './SidebarNavLink';
+
+const styles = theme => ({
+  drawerHeader: theme.mixins.toolbar,
+  nested: { paddingLeft: theme.spacing.unit * 4 },
+});
+
+/**
+ * The content of the IAR application side bar.
  */
-const Sidebar = () => (
-  <Drawer variant="permanent">
-    <img src={logo} className="App-logo" alt="logo" />
-    <h3>Assets</h3>
-    <ul>
-      <li><Link to="/assets/dept">My department</Link></li>
-      <li><Link to="/assets/all">All</Link></li>
-      <hr/>
-      <li><Link to="/help">Help</Link></li>
-      <hr/>
-      <li><LogoutLink>Sign out</LogoutLink></li>
-    </ul>
-  </Drawer>
+const Sidebar = ({ classes, history, logout }) => (
+  <div>
+    <div className={classes.drawerHeader}>
+      <Toolbar>
+        <Typography variant='title' color='inherit'>IAR</Typography>
```

Comment:
IAR text was a placeholder in the prototype. It's been decided to use the University logo here instead. This logo should also provide a link back to home, in our case, the asset department list.